packages/components/src/date-time/date.js

```diff
@@ -2,29 +2,82 @@
  * External dependencies
  */
 import moment from 'moment';
+import classnames from 'classnames';
+
 // react-dates doesn't tree-shake correctly, so we import from the individual
 // component here, to avoid including too much of the library
 import DayPickerSingleDateController from 'react-dates/lib/components/DayPickerSingleDateController';
```

Comment:
Probably existing behavior on trunk, but I see a deprecation warning when I click on this element:

| Schedule Click | React Warning |
|-----|-----|
| <img width="316" alt="Screen Shot 2021-04-02 at 10 02 47 AM" src="https://user-images.githubusercontent.com/1270189/113436987-a6588d80-939a-11eb-804c-c9933b056065.png"> | <img width="1153" alt="Screen Shot 2021-04-02 at 9 43 20 AM" src="https://user-images.githubusercontent.com/1270189/113436903-7315fe80-939a-11eb-88de-3aabe4b98e31.png"> |

Reply:
I think it happens in the react-dates components. Another reason to consider replacing the component, IMHO.